dl/src/test/java/com/intel/analytics/bigdl/bigdl/Parallel.java

```diff
@@ -0,0 +1,14 @@
+package com.intel.analytics.bigdl.tags;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+import static java.lang.annotation.ElementType.METHOD;
+import static java.lang.annotation.ElementType.TYPE;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+@org.scalatest.TagAnnotation
+@Retention(RUNTIME)
+@Target({METHOD, TYPE})
+public @interface Parallel {
+}
```

Comment:
Why does the file locate in the folder 'dl/src/test/java/com/intel/analytics/bigdl/bigdl/' rather than '''dl/src/test/java/com/intel/analytics/bigdl/tags' ?